modules/shared/src/main/resources/logback-included.xml

```diff
@@ -157,6 +187,10 @@
         <appender-ref ref="GOSSIP" />
         <appender-ref ref="GOSSIP_JSON" />
     </logger>
+    <logger name="PeerSelectLogger" level="INFO" additivity="false">
```

Comment:
you specified the log level to `INFO` but you log as `DEBUG`. you will see no logs in the log files